File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/bill/BIL2016R01GrdReportInfo.java
package nta.med.data.model.ihis.bill;

import java.math.BigDecimal;
import java.util.Date;

public class BIL2016R01GrdReportInfo {
	private String serviceId;
	private String serviceName;
	private String executeDept;
	private String executeDoctor;
	private String assignDept;
	private String assignDoctor;
	private Date actingDate;
	private String quantity;
	private BigDecimal sum;
	private BigDecimal discount;
	public BIL2016R01GrdReportInfo(String serviceId, String serviceName, String executeDept, String executeDoctor,
			String assignDept, String assignDoctor, Date actingDate, String quantity, BigDecimal sum,
			BigDecimal discount) {
		super();
		this.serviceId = serviceId;
		this.serviceName = serviceName;
		this.executeDept = executeDept;
		this.executeDoctor = executeDoctor;
		this.assignDept = assignDept;
		this.assignDoctor = assignDoctor;
		this.actingDate = actingDate;
		this.quantity = quantity;
		this.sum = sum;
		this.discount = discount;
	}
	public String getServiceId() {
		return serviceId;
	}
	public void setServiceId(String serviceId) {
		this.serviceId = serviceId;
	}
	public String getServiceName() {
		return serviceName;
	}
	public void setServiceName(String serviceName) {
		this.serviceName = serviceName;
	}
	public String getExecuteDept() {
		return executeDept;
	}
	public void setExecuteDept(String executeDept) {
		this.executeDept = executeDept;
	}
	public String getExecuteDoctor() {
		return executeDoctor;
	}
	public void setExecuteDoctor(String executeDoctor) {
		this.executeDoctor = executeDoctor;
	}
	public String getAssignDept() {
		return assignDept;
	}
	public void setAssignDept(String assignDept) {
		this.assignDept = assignDept;
	}
	public String getAssignDoctor() {
		return assignDoctor;
	}
	public void setAssignDoctor(String assignDoctor) {
		this.assignDoctor = assignDoctor;
	}
	public Date getActingDate() {
		return actingDate;
	}
	public void setActingDate(Date actingDate) {
		this.actingDate = actingDate;
	}
	public String getQuantity() {
		return quantity;
	}
	public void setQuantity(String quantity) {
		this.quantity = quantity;
	}
	public BigDecimal getSum() {
		return sum;
	}
	public void setSum(BigDecimal sum) {
		this.sum = sum;
	}
	public BigDecimal getDiscount() {
		return discount;
	}
	public void setDiscount(BigDecimal discount) {
		this.discount = discount;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Bill/BIL2016R01GrdReportInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Bill
{
    public class BIL2016R01GrdReportInfo
    {
        private String _serviceId;
        private String _serviceName;
        private String _executeDept;
        private String _executeDoctor;
        private String _assignDept;
        private String _assignDoctor;
        private String _actingDate;
        private String _quantity;
        private String _sum;
        private String _discount;
        private String _amountPaid;

        public String ServiceId
        {
            get { return this._serviceId; }
            set { this._serviceId = value; }
        }

        public String ServiceName
        {
            get { return this._serviceName; }
            set { this._serviceName = value; }
        }

        public String ExecuteDept
        {
            get { return this._executeDept; }
            set { this._executeDept = value; }
        }

        public String ExecuteDoctor
        {
            get { return this._executeDoctor; }
            set { this._executeDoctor = value; }
        }

        public String AssignDept
        {
            get { return this._assignDept; }
            set { this._assignDept = value; }
        }

        public String AssignDoctor
        {
            get { return this._assignDoctor; }
            set { this._assignDoctor = value; }
        }

        public String ActingDate
        {
            get { return this._actingDate; }
            set { this._actingDate = value; }
        }

        public String Quantity
        {
            get { return this._quantity; }
            set { this._quantity = value; }
        }

        public String Sum
        {
            get { return this._sum; }
            set { this._sum = value; }
        }

        public String Discount
        {
            get { return this._discount; }
            set { this._discount = value; }
        }

        public String AmountPaid
        {
            get { return this._amountPaid; }
            set { this._amountPaid = value; }
        }

        public BIL2016R01GrdReportInfo() { }

        public BIL2016R01GrdReportInfo(String serviceId, String serviceName, String executeDept, String executeDoctor, String assignDept, String assignDoctor, String actingDate, String quantity, String sum, String discount, String amountPaid)
        {
            this._serviceId = serviceId;
            this._serviceName = serviceName;
            this._executeDept = executeDept;
            this._executeDoctor = executeDoctor;
            this._assignDept = assignDept;
            this._assignDoctor = assignDoctor;
            this._actingDate = actingDate;
            this._quantity = quantity;
            this._sum = sum;
            this._discount = discount;
            this._amountPaid = amountPaid;
        }

    }
}